src/app/test/events/init.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+import pytest
+from mock import Mock
+
+from beer_garden import config
+from beer_garden.db.mongo.models import CommandPublishingBlackList
+from beer_garden.events import event_blacklisted
+
+
+@pytest.fixture
+def model_mocks(monkeypatch):
```

Comment:
I think I commented on this in my original review, but is there a reason to use a `Mock()` here as opposed to just using the `Event` model?